src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -68,45 +60,52 @@
+          <!-- Build the debuggee for 3.0 but run it on 3.1 -->
           <BuildProjectFramework>netcoreapp3.0</BuildProjectFramework>
           <BuildProjectMicrosoftNetCoreAppVersion>$(RuntimeVersion30)</BuildProjectMicrosoftNetCoreAppVersion>
-          <RuntimeFrameworkVersion>$(RuntimeVersionLatest)</RuntimeFrameworkVersion>
+          <RuntimeFrameworkVersion>$(RuntimeVersion31)</RuntimeFrameworkVersion>
         </Option>
         <Option Condition="'$(DailyTest)' != 'true'">
-          <BuildProjectFramework>netcoreapp2.1</BuildProjectFramework>
-          <BuildProjectMicrosoftNetCoreAppVersion>$(RuntimeVersion21)</BuildProjectMicrosoftNetCoreAppVersion>
-          <RuntimeFrameworkVersion>$(RuntimeVersion21)</RuntimeFrameworkVersion>
+          <BuildProjectFramework>netcoreapp3.0</BuildProjectFramework>
+          <BuildProjectMicrosoftNetCoreAppVersion>$(RuntimeVersion30)</BuildProjectMicrosoftNetCoreAppVersion>
+          <RuntimeFrameworkVersion>$(RuntimeVersion30)</RuntimeFrameworkVersion>
         </Option>
         <Option Condition="'$(DailyTest)' != 'true'">
-          <!-- The debuggee built for 2.1 but run it on 2.2 -->
           <BuildProjectFramework>netcoreapp2.1</BuildProjectFramework>
           <BuildProjectMicrosoftNetCoreAppVersion>$(RuntimeVersion21)</BuildProjectMicrosoftNetCoreAppVersion>
-          <RuntimeFrameworkVersion>$(RuntimeVersion22)</RuntimeFrameworkVersion>
-        </Option>
-        <Option Condition="Exists('$(RepoRootDir)/.dotnet/shared/Microsoft.NETCore.App/$(RuntimeVersion11)')">
-          <BuildProjectFramework>netcoreapp1.1</BuildProjectFramework>
-          <BuildProjectMicrosoftNetCoreAppVersion>$(RuntimeVersion11)</BuildProjectMicrosoftNetCoreAppVersion>
-          <RuntimeFrameworkVersion>$(RuntimeVersion11)</RuntimeFrameworkVersion>
-          <!-- createdump doesn't exists in 1.1 -->
-          <GenerateDumpWithGDB>true</GenerateDumpWithGDB>
-          <!-- SOS needs at least 2.0 to run. The default without this is to use the runtime being debuggged to host SOS.NETCore -->
-          <SOSHostRuntime>$(RepoRootDir)/.dotnet/shared/Microsoft.NETCore.App/$(RuntimeVersion21)</SOSHostRuntime>
+          <RuntimeFrameworkVersion>$(RuntimeVersion21)</RuntimeFrameworkVersion>
         </Option>
       </Options>
     </Option>
     <!--
-        SOS.WebApp3 (runs only on 3.0 aspnetcore)
+        SOS.WebApp3 (runs on 3.0, 3.1 and latest aspnetcore)
       -->
     <Option>
       <TestName>SOS.WebApp3</TestName>
-      <BuildProjectFramework>netcoreapp3.0</BuildProjectFramework>
-      <RuntimeFrameworkVersion>$(AspNetCoreVersion30)</RuntimeFrameworkVersion>
+      <Options>
+        <!-- Build the debuggee for 3.0 but run it on latest -->
+        <!--
+        <Option>
+          <BuildProjectFramework>netcoreapp3.0</BuildProjectFramework>
+          <RuntimeFrameworkVersion>$(AspNetCoreVersionLatest)</RuntimeFrameworkVersion>
```

Comment:
This property is something that should be used sparingly. It works well enough for console apps, but then ASP will use whatever runtime it is designed to use. If you've tested to see what bits are getting used on a daily build when all runtimes can be found by the host, feel free to dismiss this.

Reply:
What property? Are talking about that we build on 3.0 but test on 5.0? We don't have a 5.0 SDK I don't think yet. I'm probably misunderstanding your comment.

Reply:
Sorry, thought this was `RuntimeFrameworkVersion` in a csproj that forces a runtime to use. The problem with building as a netcoreapp3.0 and expect it to run as a 5.0 is that there's the possibility of the host policy kicking back and still trying to use 3.0 bits if it can find them. Given you properly install all of them, that can happen. It'd be something to test that we are running the bits we think we are here more than anything.